src/react/components/_finances/FinancialResources/SummerSessions.js

```diff
@@ -63,18 +62,17 @@ const SummerSessions = ({ links, getLink, expanded, sirStatus }) => {
                 </li>
               )}
             </HasAccessTo>
-            {/* Summer Estimator is available to users who have an UGRD Sir, 
+            {/* Summer Estimator is available to users who have an UGRD Sir,
                 but may not have any other attributes required to access the link */}
-            {isUndergradSir && links && (
+            {isUndergradSir && links ? (
```

Comment:
@marcpechaitis - Oh that's cool. I didn't know you could use a ternary conditional to make one thing or another display

Reply:
yeah in the original PR @developish suggested using the ternary as an alternative... and since I was already under the hood I figured why not do it now?